test/app_signals/resources/metrics/client_producer.json

```diff
@@ -34,8 +34,8 @@
           "metrics": [
             {
               "name": "Error",
-              "unit": "Milliseconds",
-              "sum": {
+              "unit": "",
```

Comment:
Why did we have Milliseconds as a unit before but not now ?

Reply:
The units were incorrect before, there are no units for app signal error/fault